Add call for interns (#90)
OpenScienceLabs/opensciencelabs.github.io: pages/blog/call-for-interns/index.md

---

> title: Call for interns

Reviewer: ```suggestion
title: Call for Interns 2024-01
```

> slug: call-for-interns

Reviewer: ```suggestion
slug: call-for-interns-2024-01
```

    color: #fff;
  }

  td {
    background-color: #d8f3dc; /* Color del resto de las filas */
  }

  tr:hover td {
    background-color: #f2f2f2;
  }

  pre {
    background-color: #f8f8f8;
    padding: 10px;
    border: 1px solid #ddd;
    overflow-x: auto;
  }

  a {
    color: #3498db;
    text-decoration: none;
  }

  a:hover {
    text-decoration: underline;
  }
</style>

<!-- # Intro -->

The Open Science Labs (OSL) has announced its Internship and Learning Program for the first cycle of 2024, in collaboration with The GRAPH Network. The program offers opportunities for mentors and student/collaborators, although it is unpaid. Project proposals must be open source, have a Code of Conduct, define clear tasks for interns, and have at least one mentor.

## Initial Guidelines for Internship Program Candidates:
---

Candidates are encouraged to thoroughly explore project options aligning with their skills and interests. This involves a careful review of project ideas and documentation.

Once a candidate identifies a preferred project, they should initiate contact with the mentor via email. The email should include:

* Personal Introduction
* Curriculum Vitae
* The project name

> * Detailed explanation of the project idea

Reviewer: ```suggestion
* Project idea name
```

* Motivation for wanting to collaborate on that specific project.

Additionally, candidates must confirm their availability for the entire 3-month duration of the project.

Upon mentor acceptance, candidates will be assigned preliminary tasks, such as documentation updates and bug fixes. This step enables the mentor to assess the candidate's capability to handle the project tasks effectively.

Successful completion of these initial assignments will prompt the mentor to guide the candidate in applying for the Open Science Labs Internship Program through the provided LINK.

Candidates are reminded of the importance of having prior knowledge of Git. Additionally, candidates should join the OSL Discord to stay updated on announcements related to the Internship Program.

> Recognizing the value of skill enhancement, Open Science Labs organizes study groups that candidates are encouraged to participate in, further refining their abilities.

Reviewer: ```suggestion
Recognizing the value of skill enhancement, Open Science Labs organizes study groups that candidates are encouraged to participate in, further refining their abilities. If you want to join the study group, please ask for that in the general channel.
```

## List of Participating Projects
---

Below is the list of projects participating in the current internship cycle. Each project includes key details to help candidates understand the scope and requirements.

### ArtBox

- **Description:** ArtBox is a tool set for handling multimedia files with a bunch of useful functions.
- **Category:** Multimedia Processing.
- **Organization/Project Webpage URL:** [https://osl-incubator.github.io/artbox/](https://osl-incubator.github.io/artbox/)
- **Contact:** Ivan Ogasawara [ivan.ogasawara@gmail.com](mailto:ivan.ogasawara@gmail.com)
- **Project Ideas URL:** [https://github.com/osl-incubator/artbox/issues/10](https://github.com/osl-incubator/artbox/issues/10)
- **Application Record:** [https://github.com/OpenScienceLabs/request-forms/issues/48](https://github.com/OpenScienceLabs/request-forms/issues/48)

### ArxLang/ASTx

- **Description:** ASTx is an agnostic expression structure for AST. It is agnostic because it is not specific to any language, neither to the ArxLang project, although its main focus is to provide all needed feature for ArxLang.

> - **Category:** Language Agnostic AST (Abstract Syntax Tree)

Reviewer: ```suggestion
- **Categories:** AST, Compiler
```

- **Organization/Project Webpage URL:** [https://arxlang.github.io/astx/](https://arxlang.github.io/astx/)
- **Contact:** Ivan Ogasawara [ivan.ogasawara@gmail.com](mailto:ivan.ogasawara@gmail.com)
- **Project Ideas URL:** [https://github.com/arxlang/astx/issues/21](https://github.com/arxlang/astx/issues/21)
- **Application Record:** [https://github.com/OpenScienceLabs/request-forms/issues/49](https://github.com/OpenScienceLabs/request-forms/issues/49)

### Envers

- **Description:** Envers is a command-line tool (CLI) designed to manage and version environment variables for different deployment stages such as staging, development, and production. It provides a secure and organized way to handle environment-specific configurations.

> - **Category:** DevOps/Configuration Management

Reviewer: ```suggestion
- **Categories:** DevOps, Environment Management
```

- **Organization/Project Webpage URL:** [https://osl-incubator.github.io/envers/](https://osl-incubator.github.io/envers/)
- **Contact:** Ivan Ogasawara [ivan.ogasawara@gmail.com](mailto:ivan.ogasawara@gmail.com)
- **Project Ideas URL:** [https://github.com/osl-incubator/envers/issues/18](https://github.com/osl-incubator/envers/issues/18)
- **Application Record:** [https://github.com/OpenScienceLabs/request-forms/issues/46](https://github.com/OpenScienceLabs/request-forms/issues/46)

### fqlearn

- **Description:** This Project aims to facilitate the teaching of unit operations and thermodynamics.

> - **Category:** Education/Chemical Engineering

Reviewer: ```suggestion
- **Categories:** Mathematical Modeling, Educational
```

- **Organization/Project Webpage URL:** [https://osl-pocs.github.io/fqlearn/](https://osl-pocs.github.io/fqlearn/)
- **Contact:** John Ever Vino Duran [evervino00@gmail.com](mailto:evervino00@gmail.com)
- **Project Ideas URL:** [https://github.com/osl-pocs/fqlearn/issues/30](https://github.com/osl-pocs/fqlearn/issues/30)
- **Application Record:** [https://github.com/OpenScienceLabs/request-forms/issues/52](https://github.com/OpenScienceLabs/request-forms/issues/52)

### Makim

- **Description:** Makim (or makim) is based on make and focus on improve the way to define targets and dependencies. Instead of using the Makefile format, it uses yaml format.

> - **Category:** Build Automation/Configuration Management

Reviewer: ```suggestion
- **Categories:** DevOps, Automation
```

- **Organization/Project Webpage URL:** [https://osl-incubator.github.io/makim/](https://osl-incubator.github.io/makim/)
- **Contact:** Ivan Ogasawara [ivan.ogasawara@gmail.com](mailto:ivan.ogasawara@gmail.com)
- **Project Ideas URL:** [https://github.com/osl-incubator/makim/issues/74](https://github.com/osl-incubator/makim/issues/74)
- **Application Record:** [https://github.com/OpenScienceLabs/request-forms/issues/44](https://github.com/OpenScienceLabs/request-forms/issues/44)

### noWorkflow

- **Description:** The noWorkflow project aims at allowing scientists to benefit from provenance data analysis even when they don't use a workflow system. It transparently collects provenance from Python scripts and notebooks and provide tools to support the analysis and management of the provenance.

> - **Category:** Research Provenance/Scientific Data Analysis

Reviewer: ```suggestion
- **Categories:** Provenance, Software Engineering
```

- **Organization/Project Webpage URL:** [https://gems-uff.github.io/noworkflow/](https://gems-uff.github.io/noworkflow/)
- **Contact:** João Felipe Nicolaci Pimentel [joaofelipenp@gmail.com](mailto:joaofelipenp@gmail.com)
- **Project Ideas URL:** [https://gist.github.com/JoaoFelipe/ce4cb232deb2c71d4f39afc5cbeefe2b](https://gist.github.com/JoaoFelipe/ce4cb232deb2c71d4f39afc5cbeefe2b)
- **Application Record:** [https://github.com/OpenScienceLabs/request-forms/issues/51](https://github.com/OpenScienceLabs/request-forms/issues/51)

### OSL Web Page

- **Description:** OpenScienceLabs web page, is a project that serves as a way to present OSL to the world through a web page.

> - **Category:** Web Development/Online Presence

Reviewer: ```suggestion
- **Category:** Web Development
```

- **Organization/Project Webpage URL:** [https://opensciencelabs.org//](https://opensciencelabs.org/)
- **Contact:** John Ever Vino Duran [evervino00@gmail.com](mailto:evervino00@gmail.com)
- **Project Ideas URL:** [https://github.com/osl-pocs/fqlearn/issues/84](https://github.com/osl-pocs/fqlearn/issues/84)
- **Application Record:** [https://github.com/OpenScienceLabs/request-forms/issues/50](https://github.com/OpenScienceLabs/request-forms/issues/50)

### PyDataStructs

- **Description:** PyDataStructs project aims to be a Python package for various data structures and algorithms (including their parallel implementations).

> - **Category:** Data Structures and Algorithms

Reviewer: ```suggestion
- **Categories:** Data Structures, Algorithms
```

- **Organization/Project Webpage URL:** [https://pydatastructs.readthedocs.io/en/latest/](https://pydatastructs.readthedocs.io/en/latest/)
- **Contact:** Gagandeep Singh [gdp.1807@gmail.com](mailto:gdp.1807@gmail.com)
- **Project Ideas URL:** [https://github.com/codezonediitj/pydatastructs/wiki/Planned-Features-for-v1.0.1](https://github.com/codezonediitj/pydatastructs/wiki/Planned-Features-for-v1.0.1)
- **Application Record:** [https://github.com/OpenScienceLabs/request-forms/issues/43](https://github.com/OpenScienceLabs/request-forms/issues/43)

### SciCookie

- **Description:** SciCookie is a template developed by [OpenScienceLabs](https://opensciencelabs.org/) that creates projects from project templates.

> - **Category:** Project Templates/Development Automation

Reviewer: ```suggestion
- **Category:** Project Templates, Scientific Software
```

- **Organization/Project Webpage URL:** [https://osl-incubator.github.io/scicookie](https://osl-incubator.github.io/scicookie)
- **Contact:** Ivan Ogasawara [ivan.ogasawara@gmail.com](mailto:ivan.ogasawara@gmail.com)
- **Project Ideas URL:** [https://github.com/osl-incubator/scicookie/issues/192](https://github.com/osl-incubator/scicookie/issues/192)
- **Application Record:** [https://github.com/OpenScienceLabs/request-forms/issues/47](https://github.com/OpenScienceLabs/request-forms/issues/47)

### Sugar

- **Description:** Sugar aims to organize your stack of containers, gathering some useful scripts and keeping this information centralized in a configuration file. So the command line would be very simple.

> - **Category:** Container Management/Configuration

Reviewer: ```suggestion
- **Categories:** DevOps, Container Management
```

  }

  th, td {
    border: 1px solid #ddd;
    padding: 12px;
    text-align: left;
    transition: background-color 0.3s;
  }

  th {
    background-color: #74c69d; /* Color del título */
    color: #fff;
  }

  td {
    background-color: #d8f3dc; /* Color del resto de las filas */
  }

  tr:hover td {
    background-color: #f2f2f2;
  }

  blockquote {
    border-left: 3px solid #ddd;
    margin: 15px 0;
    padding-left: 15px;
    color: #555;
  }

  code {
    background-color: #f4f4f4;
    padding: 2px 6px;
    border: 1px solid #ddd;
  }

  pre {
    background-color: #f8f8f8;
    padding: 10px;
    border: 1px solid #ddd;
    overflow-x: auto;
  }

  a {
    color: #3498db;
    text-decoration: none;
  }

  a:hover {
    text-decoration: underline;
  }
</style>

<!-- # Intro -->

The Open Science Labs (OSL) has announced its Internship and Learning Program for the first cycle of 2024, in collaboration with The GRAPH Network. The program offers opportunities for mentors and student/collaborators, although it is unpaid. Project proposals must be open source, have a Code of Conduct, define clear tasks for interns, and have at least one mentor.

> ## Roles and Collaboration:

Reviewer: # Criteria

> * **Mentors:** To submit a project, individuals need to be core contributors or obtain approval from project core members. Project proposals must adhere to open source principles, include a Code of Conduct, and outline tasks for interns. The application process involves submitting a list of proposal plans, detailing minimum requirements and utilizing a template for clarity.

Reviewer: ```suggestion
```

> * **Students/Collaborators:** Open to all interested in contributing to open source projects, applicants must be at least 18 years old. Each project sets specific criteria, and a minimum weekly commitment of 20 hours is required. The internship cycle spans three months and may be extended by mutual agreement. Applications open on January 5, 2024, with the project list shared on OSL Discord, LinkedIn, and Twitter.

Reviewer: ```suggestion
Open to all interested in contributing to open source projects, applicants must be at least 18 years old. Each project sets specific criteria, and a minimum total time for the project. The internship cycle spans three months and may be extended by mutual agreement.
```

> ## Recommendations for Internship Program Candidates:

Reviewer: ```suggestion
## First Steps for the Internship Program Candidates:
```

> * **Project Engagement:** Candidates are advised to explore project options, contact mentors, and contribute before applying. Familiarity with Git, joining the OSL Discord, and participating in study groups are emphasized.

Reviewer: the candidate in his/her first contact should inform the following information:

* Curriculum
* The name of the project and the desired project idea

in the text we also need to emphasize that the candidate should be 18y+

> * **Project Engagement:** Candidates are advised to explore project options, contact mentors, and contribute before applying. Familiarity with Git, joining the OSL Discord, and participating in study groups are emphasized.

Reviewer: Candidates are advised to explore project options according to their skills and interests (read carefully the project ideas and the project documentation).

The candidates will contact the mentors for the desired project by email and should send them the following information: Personal Introduction, Curriculum, the project name, the project idea, and the motivation for collaborating in that project idea, and additionally the candidate needs to aware that he/she is available the total amount of time for the project for the period of 3 months.

For the next step, if the mentor accept the candidate, he/she will assign some small tasks to the candidate, such as documentation updates and bug fixes. that helps the mentor to ensure that the candidate is able to accomplish with the project idea tasks.

Finally, if the candidate is able to accomplish this first assignments, the mentor will ask the candidate to apply for that project to the Open Science Labs Internship Program.  This will be done via LINK

It is important that the candidates has some previous knowledge about Git and the candidate should join OSL Discord for receiving announcements about the Internship Program.

Open Science Labs also organize some study groups, so any candidates can also participate in any study group in order to sharp their skill.

> * **Technical Preparation:** Candidates should focus on accessible issues, such as documentation updates and bug fixes, before applying.

Reviewer: ```suggestion
```

> ## Applicaction Process:

Reviewer: ```suggestion
```

> * **How to Apply:** After gaining mentor approval, candidates can apply through the provided link. Communication channels vary between mentors and students, with official OSL communication through Discord.

Reviewer: ```suggestion
```

## Guidelines for Approved Interns:

* **Communication:** Proactive communication is encouraged, with frequent updates through appropriate channels. Approved interns should use OSL Blog to document their experiences.

* **Continuous Learning:** Interns are advised to study project technologies, participate in study groups, and regularly write about their experiences on the OSL Blog.

* **Evaluations:** The internship includes midterm and final evaluations, allowing mentors to assess progress and students/collaborators to evaluate their contributions.

>**Information:** 
For more details about internships, you can [click here](https://opensciencelabs.org/programs/internship/cycles/2024-01/#osl-web-page)

## Timeline

| Date| Activity |
| ------------------------ | --------------------------------------------------------------------------------------------|

> | **December 1, 2023**     | Call for Mentoring Organizations and Projects.                                               |

Reviewer: ```suggestion
```

> | **January 3, 2024**      | Deadline for Mentoring Organizations to apply.                                               |

Reviewer: ```suggestion
```

> | **January 5, 2024**      | Announcement of selected Mentoring Organizations.                                            |

Reviewer: ```suggestion
```

| **May 20, 2024**         | Final Evaluation.                                                                            |
| **May 27-31, 2024**      | Interns present their work.                                                                  |
| **June 3, 2024**         | Official End Date; Certification process begins.                                             |

## List of Participating Projects

Below is the list of projects participating in the current internship cycle. Each project includes key details to help candidates understand the scope and requirements.

<div style="border: 2px solid #caf0f8; border-radius: 10px; padding: 15px; margin-bottom: 20px; background-color: #caf0f8;">

### ArtBox

- **Description:** ArtBox is a tool set for handling multimedia files with a bunch of useful functions.

- **Organization/Project Webpage URL:** [https://osl-incubator.github.io/artbox/](https://osl-incubator.github.io/artbox/)

- **Contact:** Ivan Ogasawara [ivan.ogasawara@gmail.com](mailto:ivan.ogasawara@gmail.com)

- **Project Ideas URL:** [https://github.com/osl-incubator/artbox/issues/10](https://github.com/osl-incubator/artbox/issues/10)

> - **Application Record:** [https://github.com/OpenScienceLabs/request-forms/issues/48](https://github.com/OpenScienceLabs/request-forms/issues/48)

Reviewer: ```suggestion
```

</div>

<div style="border: 2px solid #caf0f8; border-radius: 10px; padding: 15px; margin-bottom: 20px; background-color: #caf0f8;">

### ArxLang/ASTx

- **Description:** ASTx is an agnostic expression structure for AST. It is agnostic because it is not specific to any language, neither to the ArxLang project, although its main focus is to provide all needed feature for ArxLang.

- **Organization/Project Webpage URL:** [https://arxlang.github.io/astx/](https://arxlang.github.io/astx/)

- **Contact:** Ivan Ogasawara [ivan.ogasawara@gmail.com](mailto:ivan.ogasawara@gmail.com)

- **Project Ideas URL:** [https://github.com/arxlang/astx/issues/21](https://github.com/arxlang/astx/issues/21)

> - **Application Record:** [https://github.com/OpenScienceLabs/request-forms/issues/49](https://github.com/OpenScienceLabs/request-forms/issues/49)

Reviewer: ```suggestion
```

</div>

<div style="border: 2px solid #caf0f8; border-radius: 10px; padding: 15px; margin-bottom: 20px; background-color: #caf0f8;">

### Envers

- **Description:** Envers is a command-line tool (CLI) designed to manage and version environment variables for different deployment stages such as staging, development, and production. It provides a secure and organized way to handle environment-specific configurations.

- **Organization/Project Webpage URL:** [https://osl-incubator.github.io/envers/](https://osl-incubator.github.io/envers/)

- **Contact:** Ivan Ogasawara [ivan.ogasawara@gmail.com](mailto:ivan.ogasawara@gmail.com)

- **Project Ideas URL:** [https://github.com/osl-incubator/envers/issues/18](https://github.com/osl-incubator/envers/issues/18)

> - **Application Record:** [https://github.com/OpenScienceLabs/request-forms/issues/46](https://github.com/OpenScienceLabs/request-forms/issues/46)

Reviewer: ```suggestion
```

</div>

<div style="border: 2px solid #caf0f8; border-radius: 10px; padding: 15px; margin-bottom: 20px; background-color: #caf0f8;">

### fqlearn

- **Description:** This Project aims to facilitate the teaching of unit operations and thermodynamics.

- **Organization/Project Webpage URL:** [https://osl-pocs.github.io/fqlearn/](https://osl-pocs.github.io/fqlearn/)

- **Contact:** John Ever Vino Duran [evervino00@gmail.com](mailto:evervino00@gmail.com)

- **Project Ideas URL:** [https://github.com/osl-pocs/fqlearn/issues/30](https://github.com/osl-pocs/fqlearn/issues/30)

> - **Application Record:** [https://github.com/OpenScienceLabs/request-forms/issues/52](https://github.com/OpenScienceLabs/request-forms/issues/52)

Reviewer: ```suggestion
```

</div>

<div style="border: 2px solid #caf0f8; border-radius: 10px; padding: 15px; margin-bottom: 20px; background-color: #caf0f8;">

### Makim

- **Description:** Makim (or makim) is based on make and focus on improve the way to define targets and dependencies. Instead of using the Makefile format, it uses yaml format.

- **Organization/Project Webpage URL:** [https://osl-incubator.github.io/makim/](https://osl-incubator.github.io/makim/)

- **Contact:** Ivan Ogasawara [ivan.ogasawara@gmail.com](mailto:ivan.ogasawara@gmail.com)

- **Project Ideas URL:** [https://github.com/osl-incubator/makim/issues/74](https://github.com/osl-incubator/makim/issues/74)

> - **Application Record:** [https://github.com/OpenScienceLabs/request-forms/issues/44](https://github.com/OpenScienceLabs/request-forms/issues/44)

Reviewer: ```suggestion
```

</div>

<div style="border: 2px solid #caf0f8; border-radius: 10px; padding: 15px; margin-bottom: 20px; background-color: #caf0f8;">

### noWorkflow

- **Description:** The noWorkflow project aims at allowing scientists to benefit from provenance data analysis even when they don't use a workflow system. It transparently collects provenance from Python scripts and notebooks and provide tools to support the analysis and management of the provenance.

- **Organization/Project Webpage URL:** [https://gems-uff.github.io/noworkflow/](https://gems-uff.github.io/noworkflow/)

- **Contact:** João Felipe Nicolaci Pimentel [joaofelipenp@gmail.com](mailto:joaofelipenp@gmail.com)

- **Project Ideas URL:** [https://gist.github.com/JoaoFelipe/ce4cb232deb2c71d4f39afc5cbeefe2b](https://gist.github.com/JoaoFelipe/ce4cb232deb2c71d4f39afc5cbeefe2b)

> - **Application Record:** [https://github.com/OpenScienceLabs/request-forms/issues/51](https://github.com/OpenScienceLabs/request-forms/issues/51)

Reviewer: ```suggestion
```

</div>

<div style="border: 2px solid #caf0f8; border-radius: 10px; padding: 15px; margin-bottom: 20px; background-color: #caf0f8;">

### OSL Web Page

- **Description:** OpenScienceLabs web page, is a project that serves as a way to present OSL to the world through a web page.

- **Organization/Project Webpage URL:** [https://opensciencelabs.org//](https://opensciencelabs.org/)

- **Contact:** John Ever Vino Duran [evervino00@gmail.com](mailto:evervino00@gmail.com)

- **Project Ideas URL:** [https://github.com/osl-pocs/fqlearn/issues/84](https://github.com/osl-pocs/fqlearn/issues/84)

> - **Application Record:** [https://github.com/OpenScienceLabs/request-forms/issues/50](https://github.com/OpenScienceLabs/request-forms/issues/50)

Reviewer: ```suggestion
```

</div>

<div style="border: 2px solid #caf0f8; border-radius: 10px; padding: 15px; margin-bottom: 20px; background-color: #caf0f8;">

### PyDataStructs

- **Description:** PyDataStructs project aims to be a Python package for various data structures and algorithms (including their parallel implementations).

- **Organization/Project Webpage URL:** [https://pydatastructs.readthedocs.io/en/latest/](https://pydatastructs.readthedocs.io/en/latest/)

- **Contact:** Gagandeep Singh [gdp.1807@gmail.com](mailto:gdp.1807@gmail.com)

- **Project Ideas URL:** [https://github.com/codezonediitj/pydatastructs/wiki/Planned-Features-for-v1.0.1](https://github.com/codezonediitj/pydatastructs/wiki/Planned-Features-for-v1.0.1)

> - **Application Record:** [https://github.com/OpenScienceLabs/request-forms/issues/43](https://github.com/OpenScienceLabs/request-forms/issues/43)

Reviewer: ```suggestion
```

</div>

<div style="border: 2px solid #caf0f8; border-radius: 10px; padding: 15px; margin-bottom: 20px; background-color: #caf0f8;">

### Makim

- **Description:** SciCookie is a template developed by [OpenScienceLabs](https://opensciencelabs.org/) that creates projects from project templates.

- **Organization/Project Webpage URL:** [https://osl-incubator.github.io/scicookie](https://osl-incubator.github.io/scicookie)

- **Contact:** Ivan Ogasawara [ivan.ogasawara@gmail.com](mailto:ivan.ogasawara@gmail.com)

- **Project Ideas URL:** [https://github.com/osl-incubator/scicookie/issues/192](https://github.com/osl-incubator/scicookie/issues/192)

> - **Application Record:** [https://github.com/OpenScienceLabs/request-forms/issues/47](https://github.com/OpenScienceLabs/request-forms/issues/47)

Reviewer: ```suggestion
```

</div>

<div style="border: 2px solid #caf0f8; border-radius: 10px; padding: 15px; margin-bottom: 20px; background-color: #caf0f8;">

### Sugar

- **Description:** Sugar aims to organize your stack of containers, gathering some useful scripts and keeping this information centralized in a configuration file. So the command line would be very simple.

- **Organization/Project Webpage URL:** [https://osl-incubator.github.io/sugar/e](https://osl-incubator.github.io/sugar/)

- **Contact:** Ivan Ogasawara [ivan.ogasawara@gmail.com](mailto:ivan.ogasawara@gmail.com)

- **Project Ideas URL:** [https://github.com/osl-incubator/sugar/issues/105](https://github.com/osl-incubator/sugar/issues/105)

> - **Application Record:** [https://github.com/OpenScienceLabs/request-forms/issues/45](https://github.com/OpenScienceLabs/request-forms/issues/45)

Reviewer: ```suggestion
```
